src/ai/models/super_effective_model.py

```diff
@@ -0,0 +1,39 @@
+from typing import *
+
+from random import shuffle, randint
+from copy import deepcopy
+from ..model import ModelInterface
+from src.classes import Player, Move, Item, Party
+from src.utils import calculate_damage_deterministic
+
+from .random_model import RandomModel
+
+
+class DamageModel(ModelInterface):
```

Comment:
rename this model to `SuperGeniusModel` and the file to `super_genius_model.py`?